rero/rero-ils
# User interface: admin pages and jinja templates

* BETTER: lighten the front page search input button color
* BETTER: resize the front page background image
* BETTER: remove unused SCSS rules or replace them by a bootstrap class
* BETTER: enhance the search results views
* BETTER: enhance the editors views
* NEW: add delete fonction in libraries and documents brief views
* FIX: fix locale for schema and forms
* add a rule for typscript files in the .editorconfig

Co-authored-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>
Signed-off-by: Igor Milhit <igor.milhit@rero.ch>

--------

# To test

- bootstrap
- setup
- display admin pages, frontpage, document detailed view...

:warning: **the admin view for searching locations should be removed!**
:warning: **I just realize that a lot of `translate` are missing...** :worried:
